docs/spec/relayer/Relayer.md

```diff
@@ -87,22 +91,15 @@ func handleEvent(ev, chainA, chainB) {
        return error
     }
 
-    // Stage 2.
+    // Stage 3.
     // Create the IBC datagrams including `ev` & verify them.
+    datagram = createDatagram(ev, chainA, chainB, installedHeight)
```

Comment:
I think this used to be `pendingDatagram`, there is still a reference to this above, could you replace it with `createDatagram`?

Reply:
I think we will also need to review again now that the app state queries return the poof height already incremented.
For example in these lines further up (don't know of a way to comment on lines that are not part of the diffs):
```
   // Verify the result of the query
   sh = dest.lc.get_header(membershipProof.Height + 1)
```

Reply:
Wasn't aware this was already decided/changed. I will correct this.